### ISSUE TITLE
達成方法集の原文に問題があると思われる箇所について、訳注を付与する

### ISSUE THREAD

**Issue by momdo** [waic/wcag20#153]
Slackのtukkomiチャンネルの続き。
Wiki [Techniques Understandingへの原文へのコメント案](https://github.com/waic/wcag20/wiki/Techniques-Understanding%E3%81%B8%E3%81%AE%E5%8E%9F%E6%96%87%E3%81%B8%E3%81%AE%E3%82%B3%E3%83%A1%E3%83%B3%E3%83%88%E6%A1%88)も参照。

W3C原文にコメントして、原文を修正することがベストですが、W3C側は現在、WCAG 2.1対応に注力しており、リソースを割けない状況にあります。
そこで、必要に応じて訳注を付与したいと考えますがいかがでしょうか。
（4月ミーティングで反対意見がなければ、個別にissueを立てるつもりです。）

**Comment by momdo** [comment]
4月ミーティングで方針は了解された。waic.jp更新作業以降に作業を進める予定。

**Comment by momdo** [comment]
9月末現在では、[w3c/wcag](https://github.com/w3c) でWCAG 2.1 Understanding/Techniquesへのissueやプルリクエストを受け付けつつある模様。コメントするなら今がチャンスか。

**Comment by momdo** [comment]
リンク切れ以外は一通り訳注を付与することができたはず。
リンク切れについては別途issueを立てておく
